v0.0.1/admin-schema.json

```
@@ -0,0 +1,633 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/admin-schema.json",
+    "title": "Hub administrative settings",
+    "description": "This JSON file provides a schema for modeling hub administrative information.",
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "type": "string",
+            "format": "uri"
+        },
+        "name": {
+            "description": "The name of the hub.",
+            "type": "string",
+            "examples": "US COVID-19 Forecast Hub​"
+        },
+        "maintainer": {
+            "description": "The entity that maintains and runs the hub.",
+            "type": "string",
+            "examples": "The Consortium of Infectious Disease Modeling Hubs"
+        },
+        "contact": {
+            "description": "The name and email of a human being who serves as a point of contact for the hub.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "email": {
+                    "type": "string",
+                    "format": "email"
+                }
+            }
+        },
+        "repository_host": {
+            "description": "The name of the web host for the hub repository",
+            "type": "string",
+            "enum": [
+                "github"
```

Not clear to me what this is: does this mean that github is the only valid response here (currently)? If so, maybe we should make it `GitHub` as the "name" of the repo?
```suggestion
                "GitHub"
```